service/tbc/tbc.go

```diff
@@ -2734,7 +2920,10 @@ func (s *Server) Run(pctx context.Context) error {
 		return fmt.Errorf("df: %w", err)
 	}
 	if df != 0 {
-		blockPerDay := uint64(24 * time.Hour / s.chainParams.TargetTimePerBlock)
+		if s.g.chain == nil {
+			panic("gfy")
+		}
```

Comment:
Was this temporary or for debugging? If not, we should use a useful error message here.